examples/distributed-solver/distributed-solver.cpp

```diff
@@ -216,7 +216,7 @@ int main(int argc, char* argv[])
     auto Ainv =
         solver::build()
             .with_criteria(
-                gko::stop::Iteration::build().with_max_iters(100u).on(exec),
+                gko::stop::Iteration::build().with_max_iters(num_rows).on(exec),
```

Comment:
num_rows could easily be a million for a curious user, I don't think it's a good worst-case iteration cutoff. Do we need to change the value at all?

Reply:
The problem I see is that the user is allowed to increase the problem size. And even a moderate increase in problem size means that the solver will not converge because it hits the 100 iteration limit. 

Maybe what I will do is have the default as 1000, but allow the user to set the number of iterations as a input parameter.

Reply:
This is a great resolution to the issue, thanks!